app/views/layouts/base.html.slim

```diff
@@ -52,7 +52,8 @@ html lang="#{I18n.locale}" class='no-js'
     .site-wrap
       = render 'shared/i18n_mode' if FeatureManagement.enable_i18n_mode?
       = render 'shared/no_pii_banner' if FeatureManagement.no_pii_mode?
-      = render 'shared/usa_banner'
+      = render 'shared/fake_banner' if FeatureManagement.no_pii_mode?
+      = render 'shared/usa_banner' if !(FeatureManagement.no_pii_mode?)
```

Comment:
What do you think about combining these conditionals? Maybe something like this:
```ruby
- if FeatureManagement.no_pii_mode?
  = render 'shared/no_pii_banner'
  = render 'shared/fake_banner'
- else
  = render 'shared/usa_banner'
```
And then going a step further, what do you think about combining the `no_pii_banner` into the `fake_banner` by taking the contents of the former and adding it to the latter, then deleting the former, since those 2 will always be shown together? And finally, perhaps renaming `no_pii_mode?` to `fake_mode?` or `fake_banner_mode?`?

Reply:
Done